frontend/test/__support__/commands.js

```diff
@@ -134,6 +133,42 @@ Cypress.Commands.add("updateCollectionGraph", (groupsCollectionObject = {}) => {
   );
 });
 
+Cypress.Commands.add(
+  "sandboxTable",
+  ({
+    attribute_remappings = {},
+    card_id = null,
+    group_id = 4,
+    table_id = 2,
+  } = {}) => {
+    // Extract the name of the table, as well as `schema` and `db_id` that we'll need later on for `cy.updatePermissionsSchemas()`
+    cy.request("GET", "/api/table").then(({ body: tables }) => {
+      const [{ name, schema, db_id }] = tables.filter(table => {
+        return table.id === table_id;
+      });
+      const [attr] = Object.keys(attribute_remappings);
+
+      cy.log(`Sandbox "${name}" table on "${attr}" attribute`);
```

Comment:
This will be wrong if there are multiple, right? Will we ever do that? Maybe `Object.keys(attribute_remappings).join(", ")` just in case.

Reply:
There should never be multiple `attribute_remappings`. We agreed to pass only one sandbox/table per command.

Reply:
Actually, you're right. We CAN sandbox a single table with multiple user attributes. We didn't to it so far, but it doesn't mean we won't in the future. Great catch!